### PR TITLE
Windows scripting

### PR DESCRIPTION
**Pull request by kfrn**
I'm a PowerShell novice so I welcome others to comment :)

### CODE REVIEW

**Comment by retokromer** [comment]
Thank you, @kfrn, for this! I’ll try to check during the next days.

In my opinion, we should indicate somewhere that the new capabilities of Bash/Ubuntu on Windows 10 do actually often allow an one-to-one cooking following the macOS/Linux recipe. I was working on this – and on [vrecord](https://github.com/amiaopensource/vrecord) and [LTOpers](https://github.com/amiaopensource/ltopers) as well – for a workshop at the next SEAPAVAA conference, but sadly the workshop has not been accepted and therefore this cannot longer be a high priority on my side. (Sometime I have also to work for my living…)

**Comment by kfrn** [comment]
Good point - in fact, I generally use Bash for Windows if I'm not working on my Linux computer. I'll add a note about this.

I've never been to a SEAPAVAA conference, it would be cool to get there one day.

**Comment by retokromer** [comment]
I’ll test the PowerShell solution soon on our Windows test machine.

Regarding SEAPAVAA, I’m invited at the beginning of April in Manila to give the keynote speech on [_Imagining an AV Archive of the Future_](http://seapavaaconference.com/) – yet cannot hold a digital workshop on the very same topic ;-)

**Comment by kfrn** [comment]
Thanks @retokromer, let me know whether it goes smoothly on your end. :)
 
Congrats on the keynote spot! It's a shame you can't give the workshop on open-source AV tools, though. So many Pasifika archives are poorly funded and could surely make good use of them. I see that there is a workshop on personal digital archiving, at least.

**Comment by kfrn** [comment]
Also, @ablwr, you might want to check the CSS file. I added some styling for my codeblock, it may be kind of janky.

File: css/css.css [hunk 1]
```diff
@@ -71,3 +71,12 @@ div {
   -webkit-transform: scale(1.3) translateZ(0);
   transform: scale(1.3) translateZ(0);
 }
+
+.codeblock {
+  font-family: monospace
+  padding:2px 4px;
```
Comment (retokromer) [inline css/css.css:77]: I guess it’s good practice to add a space after the colon for readability (also below).

**Comment by retokromer** [comment]
Apart my remark on CSS, LGTM.

**Comment by ablwr** [comment]
Already late for work so I don't have time to double-check the CSS but I trust my fellows and we can adjust later!

**Comment by retokromer** [comment]
OK, then I merge.

**Comment by retokromer** [comment]
Thank you, @kfrn!

File: css/css.css (hunk 1 above)
Comment (kfrn) [reply css/css.css:77]: Oops! I confess that I grabbed these right from the minified bootstrap file. Thanks for fixing it in your housekeeping commit.

Comment (retokromer) [reply css/css.css:77]: No problem at all! It’s just easier for maintenance, in my opinion.

**Pull request merged**